content/TrueCommand/TCGettingStarted/InitialConfig.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Creating the Initial Configuration"
+description: "This article describes how to initially configure TrueCommand."
+aliases:
+weight: 20
+tags:
+- tcinstall
+- tcadmin
+- tcconfig
+- tcresetpwd
+- scaletoptoolbar
+- coretoptoolbar
+---
+{{< toc >}}
+TrueCommand allows you to manage all your TrueNAS systems from one location. This multi-system management increases efficiency and simplifes operations. To get started with the initial configuration of TrueCommand, ensure that the following steps have been completed:
+
+* You have created the administrator account. See [Creating the Admin Account]({{< relref "/TrueCommand/TCGettingStarted/AdminAccounts.md" >}}) for more information.
+* You have created the necessary user accounts. See [Creating User Accounts]({{< relref "/TrueCommand/TCGettingStarted/UserAccounts.md" >}}).
+* You familiar with the interface toolbars and icons. Review them here [Interface Overview]({{< relref "/TrueCommand/TCGettingStarted/InterfaceOverview.md" >}}).
```

Comment:
this is not necessarily a step, but we can make it one
```suggestion
3. Become familiar with the interface toolbars and icons. Review them here [Interface Overview]({{< relref "/TrueCommand/TCGettingStarted/InterfaceOverview.md" >}}).
```

Reply:
Change made.